examples/rust/Earthfile

```
@@ -0,0 +1,44 @@
+FROM rust:1.49
+WORKDIR /example-rust
+
+plan:
+    RUN cargo install cargo-chef --version 0.1.11
+    
+    # Compute a lock-like file for our project
+    RUN cargo chef prepare --recipe-path recipe.json
+    SAVE ARTIFACT recipe.json recipe.json
+deps:
```

Some white-space for consistency.

```suggestion
    SAVE ARTIFACT recipe.json recipe.json

deps:
```